.github/workflows/release.yml

```diff
@@ -746,9 +746,33 @@ jobs:
       with:
         arch: ${{ matrix.host }}
 
+    - name: Save code signing certificate to disk
+      id: save_cert
+      uses: timheuer/base64-to-file@v1
+      with:
+        filename: 'openttd.p12'
+        # The certificates in a PKCS12 file encoded as a base64 string
+        encodedString: ${{ secrets.WIN_CERTIFICATE_P12 }}
```

Comment:
Can you place call this `WINDOWS_CERTIFICATE_P12`? Being a bit more verbose here won't hurt anyone :)